### DISCUSSION TITLE
Question regarding FOB and Rally Point

### DISCUSSION THREAD

**Discussion by XDSenDX** [Vdauphin/HeartsAndMinds#1566] [Question and answer]
For the fob is there any ways to move it? And for rally point i can't find the Zeus rally point backpack in arsenal.

Cheers.

**Comment by Vdauphin** [comment]
Hello,

http://vdauphin.github.io/HeartsAndMinds/InGame-documentation#respawn-position

> And for rally point i can't find the Zeus rally point backpack in arsenal.

There are in my Arsenal

Cheers

**Reply by XDSenDX** [reply]
I understood the FOB now

> There are in my Arsenal

Can you tell me the name of it cuz i still can't find it?

Cheers.

**Reply by Vdauphin** [reply]
> Can you tell me the name of it cuz i still can't find it?

Lazy to open A3 now, ask to A3 discord
